client/src/pages/MapPage/MapPage.js

```diff
@@ -3,15 +3,31 @@ import './MapPage.scss';
 import MapResults from '../../components/Search/MapResults/MapResults';
 import { connect } from 'react-redux';
 import fetchMonuments from '../../actions/map';
+import {zoom} from "leaflet/src/control/Control.Zoom";
 
 class MapPage extends React.Component {
 
     constructor(props) {
         super(props);
+        this.state = {
+            zoomSize: '5'
+        }
```

Comment:
Minor: missing `;`

Reply:
In the state, I don't think it uses a ' ; '

If there were more things in the list, then it would use a ' , ' I believe

Reply:
I was referring to line 14, after the `}`.

Reply:
Oh, sorry XD